### PR TITLE
[new release] caisar (2.1)

### PR DESCRIPTION
**Pull request by caisar-platform**
CHANGES:

- [test] Increased the test coverage of CAISAR.

- [dependencies] Updated dependencies (Why3 and Base).

As for [release 2.0](https://github.com/ocaml/opam-repository/pull/26122/files), some tests will need to be commented.

### CODE REVIEW

**Comment by mseri** [comment]
Should we disable the tests here?

**Comment by caisar-platform** [comment]
> Should we disable the tests here?

Yes please. We introduced a configurator for this release that will offer more granularity over which tests are enabled or not in the future.

File: packages/caisar/caisar.2.1/opam [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "--promote-install-files=false"
+    "@install"
+    "@runtest" {with-test}
```
Comment (mseri) [inline packages/caisar/caisar.2.1/opam:49]:
```suggestion
```

**Comment by mseri** [comment]
Thanks

**Pull request merged**